docker/server-v2/Dockerfile

```diff
@@ -1,15 +1,10 @@
-# For using private GitHub dependencies, CI downdloads all crates outside of the contatiner
-# Not expected to work locally
-
-# syntax=docker/dockerfile:experimental
 FROM rust:1.72-bookworm as builder
 RUN apt-get update && apt-get install -y linux-libc-dev liburing-dev clang && \
-    # ^ We need a newer version of `linux-libc-dev` from backports than the one installed by default
```

Comment:
This comment should be left, no?

Reply:
@perekopskiy i only removed the misleading deprecated comment, command to install the package is still there. since we've migrated to Debian Bookworm, we don't need to install it from backports (and we are not doing it anymore - you can see that we are not adding backports repo before installing this package), we have a fresh enough version in default package repo. tl;dr - we've forgotten to remove this comment, and it doesn't make any sense now